ci-operator/step-registry/openshift-qe/ocpbugs-77510-e2e-test/openshift-qe-ocpbugs-77510-e2e-test-commands.sh

```
@@ -0,0 +1,635 @@
+    
+    oc create namespace "$NAMESPACE" || {
+        log_error "Failed to create namespace $NAMESPACE"
+        return 1
+    }
+    
+    oc label namespace "$NAMESPACE" test="ocpbugs-77510-$TEST_SCALE-scale" || true
+    
+    # Create services with potential for serviceUpdateNotNeeded() bug
+    # EXACT copy of working verification script approach
+    for i in $(seq 1 $SERVICE_COUNT); do
+        oc create deployment test-app-$i --image=nginx:alpine --replicas=$PODS_PER_SERVICE -n "$NAMESPACE"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)